.env.example

```diff
@@ -1,8 +1,31 @@
-export SERVICE_SLUG=service-slug
-export SERVICE_SECRET=service-secret
+####################################
+#                                  #
+#   COMMON                         #
+#                                  #
+####################################
+
+export SERVICE_SLUG=slug
+export SERVICE_SECRET=secret
+export SERVICE_TOKEN=token
+
+####################################
+#                                  #
+#   MOCK SERVICES                  #
+#                                  #
+####################################
+
 export USER_DATASTORE_URL=http://localhost:44444
 export USER_FILESTORE_URL=http://localhost:44445
 export SUBMITTER_URL=http://localhost:44446
-export SERVICE_PATH=service-path
-export SERVICE_TOKEN=token
-export MODULE__ministryofjustice_fb_components_core='/path/to/your/form-builder/fb-components-core'
+
+####################################
+#                                  #
+#   DOCKER SERVICES                #
+#                                  #
+####################################
+
+# export USER_DATASTORE_URL=http://localhost:10001
+# export USER_FILESTORE_URL=http://localhost:10002
+# export SUBMITTER_URL=http://localhost:4080
```

Comment:
`export SUBMITTER_URL=http://localhost:10003`?

Reply:
Yeah, should be that. Huh

Reply:
See commit below